### DISCUSSION TITLE
Get Kubernetes dictionary information

### DISCUSSION THREAD

**Discussion by huyichao123456** [micronaut-projects/micronaut-core#6233] [Q&A]
### Feature description

Micronaut How to obtain Kubernetes dictionary information

**Comment by pgressa** [comment]
Hi @huyichao123456, what you mean by Kubernetes dictionary? Micronaut has quite [micronaut-kubernetes](https://github.com/micronaut-projects/micronaut-kubernetes) module that uses official k8s SDK.
